doc/rtd/conf.py

```diff
@@ -67,11 +67,12 @@
 # of the sidebar.
 html_logo = 'static/logo.png'
 
-def generate_docstring_from_schema(app, what, name, obj, options, lines):
+def generate_docstring_from_schema(*args):
```

Comment:
This currently implements similar logic as before but without the memoized function call (since the __doc__ is given the output of `get_meta_doc()` already). Since the docstring already has what it needs I don't think this preprocessing is required at all.

Reply:
Addressed in [e1b52e3b61960556](https://github.com/canonical/cloud-init/pull/1101/commits/e1b52e3b619605565311728108334e6da720c42a).